app/repos/cdo.js

```diff
@@ -298,7 +299,10 @@ const getSummaryCdos = async (filter) => {
         ]
       }
     ],
-    order: [[sequelize.col('registration.cdo_expiry'), 'ASC']]
+    order: [
```

Comment:
Not 100% sure why  'joined_exemption_scheme' is an addition orderBy? Is it that some records don't have a cdo_expiry but an exemption_scheme date?

Reply:
this is getting ready for the next story, although I don't think there are any without cdo_expiry at the moment.  Maybe I should just remove and replay this when I get to the story.